src/system.c

```diff
@@ -1063,6 +1063,12 @@ void InitSystem (
 #ifdef SYS_IS_64_BIT
     SyStorMin = 128 * 1024L;
     SyStorMax = 2048*1024L;          /* This is in kB! */
+#if defined(HAVE_SYSCONF)
+#if defined(_SC_PAGESIZE) && defined(_SC_PHYS_PAGES)
+    // Set to 3/4 of memory size (in kB)
+    SyStorMax = (sysconf(_SC_PAGESIZE) * sysconf(_SC_PHYS_PAGES) * 3L) / 4 / 1024;
```

Comment:
Your description says
> On 64-bit OSes, this switches to a default of 3/4 of physical memory, or 2GB, whichever is larger

but I don't see how it's doing the 'whichever is larger' part. Isn't this just setting it to a default of ¾ of physical memory in any case?

Reply:
That's a mistake, will fix